docs/source/configuring-slack-for-spidermon.rst

```diff
@@ -0,0 +1,31 @@
+.. _configuring-slack-bot:
+
+====================================
+Configure Slack bot for Spidermon
+====================================
+
+What are bots?
+==============
+
+A bot is a type of Slack App designed to interact with users via conversation.
+
+To work with `Slack Actions <https://spidermon.readthedocs.io/en/latest/actions.html#slack-action>`_, you will need a Slack bot which would send `notifications <https://spidermon.readthedocs.io/en/latest/getting-started.html#slack-notifications>`_ to your Slack workplace from Spidermon.
+
+Steps
+=====
+
+.. note:
+    You would need to be leader of Slack workplace for which you are trying to create a bot.
+
+1. `Create Slack bot <https://get.slack.help/hc/en-us/articles/115005265703-Create-a-bot-for-your-workspace>`_ by following the steps listed on the link.
+
+2. Once bot is installed, you can find Bot User OAuth Access Token in the settings. This `Bot User OAuth Access Token` is what we use for `SPIDERMON_SLACK_SENDER_TOKEN`.
```

Comment:
```suggestion
2. Once your bot is installed, find your `Bot User OAuth Access Token` in your bot settings. Your `Bot User OAuth Access Token` is what you use for ``SPIDERMON_SLACK_SENDER_TOKEN`` (see below).
```
- Is ‘installed’ meant to be a synonym of ‘created’? If so, it’s better to be consistent with our vocabulary. Otherwise, maybe the previous step should say “Create and install”, or there should be an intermediate step about installing.

Reply:
Very crucial. Thanks for this. 
Constant vocab ++